EIPS/eip-draft_increase_blob_base_fee.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: <to be assigned>
+title: Increase Blob Base Fee
+description: Adjust the MIN_BASE_FEE_PER_BLOB_GAS to speed up price discovery on blob space
+author: <Max Resnick (@MaxResnick)>
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians using the template in the EIPs category:
https://ethereum-magicians.org/c/eips/5